src/Roundabout/roundabout-it-container.tsx

```diff
@@ -2,8 +2,13 @@ import React from "react";
 
 export function RoundaboutItContainer({ children }: { children: React.ReactNode }) {
     return (
-        <div className="roundabout-iteration-title fr-col-12 fr-mb-3w">
-            <div className="fr-grid-row">{children}</div>
+        <div className="roundabout-iteration-title fr-col-12">
+            <div
+                className="fr-grid-row fr-border fr-py-2w"
+                style={{ borderBottom: "solid #DDDDDD 1px" }}
```

Comment:
use color from dsfr

Reply:
With the inclusion of the `<hr />` we can remove this